openquake/server/views.py

```diff
@@ -94,7 +94,7 @@
 AELO_FORM_PLACEHOLDERS = {
     'lon': 'Longitude',
     'lat': 'Latitude',
-    'vs30': 'Vs30 (default 760 m/s)',
+    'vs30': 'Vs30 (fixed at 760 m/s)',  # FIXME: (default 760 m/s)
     'siteid': 'Site name',
```

Comment:
I guess you can remove the FIXME comment now

Reply:
I added the FIXME as a reminder to put back the original string once we make the vs30 input field editable again, but I suppose we wouldn't forget anyway.